cart_wrench_dpm:
  ros__parameters:
  
    robot_base_link: base_link
    end_effector_link: flange
    wrench_topic: /robotiq_force_torque_sensor_broadcaster/wrench
    joint_names:
      - J1
      - J2
      - J3
      - J4
      - J5
      - J6
